src/presence/__tests__/presenceReducers-test.js

```diff
@@ -2,9 +2,10 @@
 import deepFreeze from 'deep-freeze';
 
 import { PRESENCE_RESPONSE, EVENT_PRESENCE, ACCOUNT_SWITCH } from '../../actionConstants';
-import presenceReducers, { activityFromPresence, timestampFromPresence } from '../presenceReducers';
+import presenceReducers, { updateUserWithPresence } from '../presenceReducers';
 
 const fiveSecsAgo = Math.floor(new Date() - 5) / 1000;
+const serverTimestamp = new Date() / 1000;
```

Comment:
how about just using const values? It is probably not an issue here, but I would try not to use non-deterministic values in tests.